### ISSUE TITLE
HSS_UPDATE fails due to wrong file name

### ISSUE THREAD

**Issue by antonbrauchle-microchip** [polarfire-soc/polarfire-soc-discovery-kit-reference-design#3]
The TCL setup file MPFS_DISCOVERY_KIT_REFERENCE_DESIGN.tcl points to a wrong file in the HSS GitHub repository.
The file "hss-envm-wrapper.mpfs-Discovery-kit-es.hex" is not available in the release folder. The correct file is "hss-envm-wrapper.mpfs-disco-kit.hex"

**Comment by hughbreslin** [comment]
This will be updated for the next release :)

**Comment by ombhilare999** [comment]
Hey hi,

Even after renaming the HSS name in the .tcl file, are you able to program the board?

I am facing the following error:
```
The Execute Script command succeeded.
Error:  The command 'run_tool' failed.
Error:  Failure when executing Tcl script. [ Line 530 ]
Error:  The Execute Script command failed.
The MPFS_DISCOVERY_I2C_LOOPBACK project was closed.
```

The .tcl script I am using which contains right hex name: https://pastebin.com/nQ6pNQrs

The steps to reproduce:
```
git clone https://github.com/polarfire-soc/polarfire-soc-discovery-kit-reference-design.git
cd polarfire-soc-discovery-kit-reference-design
libero script:MPFS_DISCOVERY_KIT_REFERENCE_DESIGN.tcl SCRIPT_ARGS:I2C_LOOPBACK+HSS_UPDATE+PROGRAM logfile:MPFS_DISCOVERY_KIT_REFERENCE_DESIGN.log
```

My aim is to update the HSS on the board, cause the following HSS which is from the release of this repo seems not working with the Linux image.
![image](https://github.com/polarfire-soc/polarfire-soc-discovery-kit-reference-design/assets/40693854/7caabe22-d717-4563-93e2-76285b547183)

Thank you.

**Comment by hughbreslin** [comment]
@ombhilare999 it doesn't look like your script is failing to program the board - the log shows a failure on line 530 which appears to be when the HSS configuration is generated. Could you try opening the Libero project and seeing the state of the design flow? As a sanity check you could try doing this manually and seeing what happens.  

I suspect the "GENERATEPROGRAMMINGDATA" stage is failing not the programming.

The HSS included in the release FPExpress job asset shouldn't have an issue booting Linux, could you add a bit more info on that issue?

**Comment by hughbreslin** [comment]
@ombhilare999 I was actually able to reproduce your issue by total accident. The fix for the HSS link has been mirrored and I was testing on a PC that doesn't have wget set up, when I ran with HSS_UPDATE I got the same failure as you as the hex file was never downloaded and didn't exist at the path specified. When I manually downloaded the hex file and put it in the script_support/ directory HSS_UPDATE passed. I'd double check it is in script_support :) But from today the change was mirrored for the download link so cloning the latest scripts and running them should pass without an issue.
